src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -0,0 +1,1050 @@
+.class public auto ansi beforefieldinit C
+	extends [mscorlib]System.Object
+{
+	// Fields
+	.field private initonly int32 '<P>k__BackingField'
+	.custom instance void [mscorlib]System.Runtime.CompilerServices.CompilerGeneratedAttribute::.ctor() = (
+		01 00 00 00
+	)
+	// Methods
+	.method public hidebysig specialname 
+		instance int32 get_P () cil managed 
+	{
+		// Method begins at RVA 0x2050
+		// Code size 7 (0x7)
+		.maxstack 8
+		IL_0000: ldarg.0
+		IL_0001: ldfld int32 C::'<P>k__BackingField'
+		IL_0006: ret
+	} // end of method C::get_P
+	.method public hidebysig specialname rtspecialname 
+		instance void .ctor () cil managed 
+	{
+		// Method begins at RVA 0x2058
+		// Code size 7 (0x7)
+		.maxstack 8
+		IL_0000: ldarg.0
+		IL_0001: call instance void [mscorlib]System.Object::.ctor()
+		IL_0006: ret
+	} // end of method C::.ctor
+	// Properties
+	.property instance int32 P()
+	{
+		.get instance int32 C::get_P()
+	}
+} // end of class C
+");
+        }
+
+        [Fact(Skip = "PROTOTYPE(semi-auto-props): Mixing semicolon-only with field not yet working.")]
+        public void TestSimpleCase()
+        {
+            var comp = CreateCompilation(@"
+public class C
+{
+    public string P { get; set => field = value; }
+}
+");
+            VerifyTypeIL(comp, "C", @"
+.class public auto ansi beforefieldinit C
+	extends [mscorlib]System.Object
+{
+	// Fields
+	.field private string '<P>k__BackingField'
+	.custom instance void [mscorlib]System.Runtime.CompilerServices.CompilerGeneratedAttribute::.ctor() = (
+		01 00 00 00
+	)
+	// Methods
+	.method public hidebysig specialname 
+		instance string get_P () cil managed 
+	{
+		.custom instance void [mscorlib]System.Runtime.CompilerServices.CompilerGeneratedAttribute::.ctor() = (
+			01 00 00 00
+		)
+		// Method begins at RVA 0x2050
+		// Code size 7 (0x7)
+		.maxstack 8
+		IL_0000: ldarg.0
+		IL_0001: ldfld string C::'<P>k__BackingField'
+		IL_0006: ret
+	} // end of method C::get_P
+	.method public hidebysig specialname 
+		instance void set_P (
+			string 'value'
+		) cil managed 
+	{
+		// Method begins at RVA 0x2058
+		// Code size 8 (0x8)
+		.maxstack 8
+		IL_0000: ldarg.0
+		IL_0001: ldarg.1
+		IL_0002: stfld string C::'<P>k__BackingField'
+		IL_0007: ret
+	} // end of method C::set_P
+	.method public hidebysig specialname rtspecialname 
+		instance void .ctor () cil managed 
+	{
+		// Method begins at RVA 0x2061
+		// Code size 7 (0x7)
+		.maxstack 8
+		IL_0000: ldarg.0
+		IL_0001: call instance void [mscorlib]System.Object::.ctor()
+		IL_0006: ret
+	} // end of method C::.ctor
+	// Properties
+	.property instance string P()
+	{
+		.get instance string C::get_P()
+		.set instance void C::set_P(string)
+	}
+} // end of class C
+");
+        }
+
+        [Fact(Skip = "PROTOTYPE(semi-auto-props): Initializer isn't added and no code is generated for it. This is because we use NonFieldBackingField when we add initializers. A cycle needs to be fixed.")]
```

Comment:
>"PROTOTYPE(semi-auto-props): Initializer isn't added and no code is generated for it. This is because we use NonFieldBackingField when we add initializers. A cycle needs to be fixed.") [](http://example.com/codeflow?start=21&length=184)

This should be reevaluated. I hope that after all feedback for symbols (up to commit 52) is addressed, the scenario will start working. Even though it is not a goal to get it working in the initial PR, it would be good to enable all passing tests anyway.

Reply:
@AlekseyTs It doesn't currently work, but for a different reason now, ie, ERR_InitializerOnNonAutoProperty. I'm updating the skip reason.